tools/vsextension/ProjectTemplates/oeenclave-linux/enclave.vstemplate

```diff
@@ -0,0 +1,38 @@
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
+  <TemplateData>
+    <Name>Open Enclave TEE Project (Linux)</Name>
+    <Description>A project file for creating an Open Enclave enclave</Description>
+    <ProjectType>VC</ProjectType>
+    <SortOrder>1000</SortOrder>
+    <CreateNewFolder>true</CreateNewFolder>
+    <DefaultName>MyEnclave</DefaultName>
+    <ProvideDefaultName>true</ProvideDefaultName>
+    <LocationField>Enabled</LocationField>
+    <EnableLocationBrowseButton>true</EnableLocationBrowseButton>
+    <Icon>__TemplateIcon.ico</Icon>
+    <PromptForSaveOnCreation>true</PromptForSaveOnCreation>
+  </TemplateData>
+  <TemplateContent>
+    <Project TargetFileName="$projectname$.vcxproj" File="MyEnclave.vcxproj" ReplaceParameters="true">
+      <ProjectItem ReplaceParameters="true" TargetFileName="$projectname$.vcxproj.filters">MyEnclave.vcxproj.filters</ProjectItem>
+      <ProjectItem ReplaceParameters="true" TargetFileName="$projectname$_ecalls.c">ecalls.c</ProjectItem>
+      <ProjectItem ReplaceParameters="true" TargetFileName="$projectname$.edl">MyEnclave.edl</ProjectItem>
+      <ProjectItem ReplaceParameters="true" TargetFileName="$projectname$.Debug.conf">MyEnclave.Debug.conf</ProjectItem>
+      <ProjectItem ReplaceParameters="true" TargetFileName="$projectname$.Release.conf">MyEnclave.Release.conf</ProjectItem>
+      <ProjectItem ReplaceParameters="true" TargetFileName="makekey.sh">makekey.sh</ProjectItem>
+    </Project>
+  </TemplateContent>
+  <WizardExtension>  
+    <Assembly>OpenEnclaveSDK, Version=1.0.0.0, Culture=Neutral, PublicKeyToken=6089df5bbfa28701</Assembly>  
+    <FullClassName>OpenEnclaveSDK.WizardImplementation</FullClassName>  
+  </WizardExtension>  
+  <WizardExtension>
+    <Assembly>NuGet.VisualStudio.Interop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</Assembly>
+    <FullClassName>NuGet.VisualStudio.TemplateWizard</FullClassName>
+  </WizardExtension>
+  <WizardData>
+    <packages repository="extension" repositoryId="OpenEnclaveVisualStudioExtension-1">
```

Comment:
Curious. Why is it Extension-1? And does the version number below have any significance?

Reply:
Historical artifact I believe.  The repository id is in the VS marketplace and subsequent updates need to retain the same value to update the released item.